docs/external-classes.json

```diff
@@ -1,98 +1,50 @@
 [{
   "name": "Google\\Auth\\",
-  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.php"
+  "uri": "https://github.com/google/google-auth-library-php/blob/master/src/%s.html"
 }, {
   "name": "Google\\GAX\\",
-  "uri": "https://github.com/googleapis/gax-php/tree/master/src/%s.php"
+  "uri": "https://googleapis.github.io/gax-php/master/Google/GAX/%s.html"
```

Comment:
Perhaps the doc generator could parse `composer.json` and get the correct version of gax to inject into the URL?